faq/index.html

```diff
@@ -0,0 +1,71 @@
+<html>
+
+<head>
+  <link rel="stylesheet" href="../css/style.min.css">
+</head>
+
+<body>
+  <section class="faq">
+    <h1>FAQ - Perguntas Frequentes</h1>
+
+    <ul>
+      <li>
+        <input type="checkbox" checked>
+        <h2>Como funcionam as inscrições?</h2>
+        <p>
+          hue
+        </p>
+      </li>
+      <li>
+        <input type="checkbox" checked>
+
+        <h2>Posso levar meu Notebook?</h2>

```

Comment:
a lot of `h2`s in the same section, I think this can be replaced by an `h3`, if necessary change de css to increase the size, color, etc